.circleci/config.yml

```diff
@@ -43,19 +44,52 @@ jobs:
            echo "export START_TIME=$(date +%Y%m%d-%Hh%Mm%Ss)" >>$BASH_ENV
            echo "export GCP_IMAGE=gcr.io/${GOOGLE_PROJECT_ID}/${IMAGE}" >> $BASH_ENV
            echo "export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )" >>$BASH_ENV
-           #echo "export HELM_TEST_NAME=$(echo $CIRCLE_BRANCH | sed 's/[^A-Za-z0-9\\-]*//g' | tr '[:upper:]' '[:lower:]')" >>$BASH_ENV
            export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )           
            echo "export HELM_TEST_NAME=pr-${PR}" >> $BASH_ENV
            sudo service apport stop
+
       - run:
-         name: Docker Build taraxa-node
-         command: |
-           docker build -t ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} .
-      - run:
-         name: Docker Build taraxa-node-ctest
+         name: Login into gcr
          command: |
-          docker build -t ${IMAGE}-ctest:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} \
-            --target build .
+            echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
+ 
+#      - run:
+#         name: Docker Build taraxa-node
+#         command: |
+#           docker build -t ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} .
+
+      - docker-cache/with-save-restore-images:
+         repository: gcr.io/${GOOGLE_PROJECT_ID}
+         images: >-
+           ${IMAGE}:${DOCKER_BRANCH_TAG}
+           ${IMAGE}-cache:${DOCKER_BRANCH_TAG}
+           ${IMAGE}-cache:latest
+           ${IMAGE}-develop:latest
+         steps:
+           - docker-cache/build:
+              command: |
+                docker build -t ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} .
+                 
+#      - run:
+#         name: Docker Build taraxa-node-ctest\
+#         command: |
+#          docker build -t ${IMAGE}-ctest:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} \
+#            --target build .
+      - docker-cache/with-save-restore-images:
```

Comment:
We should name these actions accordingly. The default name is "_DRIC: Restore cache from registry gcr.io/${GOOGLE_PROJECT_ID}_"

Reply:
hmm not sure if i can, i can name all the steps i explicitly declare , but i think the once you see with 'DRIC' are internal steps of the orb's command.

Reply:
As i expected the name is not supported on all commands check run 1447

Reply:
Tested adding name properties, but i had to roll them back , tthey're not supported.